designer/client/src/components/graph/node-modal/editors/field/Textarea.js

```diff
@@ -29,7 +29,7 @@ export default function Textarea(props) {
                         autoFocus={autoFocus}
                         readOnly={readOnly}
                         placeholder={placeholder}
-                        className={classNames([
+                        className={cx([
```

Comment:
Maybe we can add some new props(validated) to TextAreaWithFocus and inside TextAreaWithFocus add that class? Because we use that in different places